Products/PloneBooking/README.txt

```diff
@@ -40,9 +65,9 @@ How to book objects ?
 
    * Then :
 
-     - fill out the form: title (not necessary), your name and email.
+     - fill out the form:  your number, phone number and email, booking title, comment (not necessary).
```

Comment:
I see below you changed phone number to be required, but it would be have been nicer to add some kind of configuration to allow that setting to be changed without having to modify code.